.github/workflows/capsule-cypress.yml

```diff
@@ -130,7 +130,7 @@ jobs:
         working-directory: frontend-monorepo
 
       - name: Run Cypress tests
-        run: npx nx affected:e2e --record --key ${{ secrets.CYPRESS_RECORD_KEY }}
+        run: npx nx affected:e2e --record --key ${{ secrets.CYPRESS_RECORD_KEY }} --env.teardownNetworkAfterFlows=true
```

Comment:
The environment variable can be passed below with the other ones. 
But I think the default should be set to true in cypress.config.js and testers should have a local cypress.env.json file to set it to false if they want to. The default should be the same as CI in my opinion.

Reply:
Good call - Rectified!